CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fixed race condition in AWSV4SignerAuth & AWSV4SignerAsyncAuth when using refreshable credentials ([#470](https://github.com/opensearch-project/opensearch-py/pull/470))
 ### Security
 ### Dependencies
-- Bumps `sphinx` from <7.1 to <7.2
+- Bumps `sphinx` from <7.1 to <7.3
```

Comment:
Should we add a new changelog entry for this change instead of making changes to existing one?